# example of train and validate parameters setting
# it is always better to write absolute paths of files needed
# you never know from which location you launch the process

resume: false

validation_frequency: 3

train_set_size: 3970

validate_set_size: 3970

epochs: 1000

solver_config: /home/leonida/ThesisCode/LSTM_on_Create2/src/neural_network_planner/NetModels/LSTM/deep_lstm_solver.prototxt

trained_weights: /home/leonida/ThesisCode/LSTM_on_Create2/src/neural_network_planner/NetModels/LSTM/Snapshots/stack-fc/yaw/LSTM_deep-multiclass-3-64-16-27_iter_500000.caffemodel

GPU: true

folder_path: /home/leonida/ThesisCode/LSTM_on_Create2/src/neural_network_planner/NetModels/

averaged_ranges_size: 8

multiclass: true

steer_feedback: true

logs_path: /home/leonida/ThesisCode/LSTM_on_Create2/logs/ 

dist_scale: 1

batch_updates: 1

iter_size: 1

stop_straight: true
